app/views/customers/show.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1><%= @customer.name %></h1>
+
+<h2>Items</h2>
+<ul> 
+  <% @customer.items.each do |item| %>
+    <li>
+      <p>Item: <%= item.name %></p>
+      <p>Price: <%= "$#{item.price}" %></p>
+      Supermarket: <%= item.supermarket.name %>
+    </li>
+  <% end %>
+</ul>
+
+<%= form_with url: "/customers/#{@customer.id}", method: :post, data: {turbo: false} do |form| %>
```

Comment:
Building off the comment I left in your controller:
the more RESTful path for this form would be "/customers/:id/items"